.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: php
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+
+php:
+  - 7.1
+  - 7.2
+  - 7.3
+
+env:
+  - ILLUMINATE_VERSION=5.5.* TESTBENCH_VERSION=3.5.* PHPUNIT_VERSION=^6.0
```

Comment:
Currently these lines do nothing, you’re setting environment variabels but you’re not using them. I’d take the geo-routes Travis file as an example, you will see some composer require statements to install to correct versions